src/Compilers/CSharp/Test/CommandLine/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+{
+  "parallelizeAssembly": true,
```

Comment:
What are the defaults for this file?  I think we need to ensure parallelizeCollection is false.  I'd personally rather be explicit htan implicit.